configuration/scripts/tests/cice_test_codecov.csh

```diff
@@ -0,0 +1,75 @@
+#!/bin/csh 
+
+# This was a script on gordon
+# This script should only be run on hardware with the gnu compiler with a
+#   modified Macros file to turn on the codecov flags
+# This should be run interactively because git push will require login information
+
+#PBS -N cice_test
+#PBS -q standard
+#PBS -A NRLSS03755018
+#PBS -l application=Regional-Arctic-System-Model
+#PBS -l select=1:ncpus=32:mpiprocs=32
+#PBS -l walltime=24:00:00
+#PBS -j oe
+#PBS -M anthony.p.craig@gmail.com
+#PBS -m be
+
+#set verbose
+module load costinit git
+
+set scrdir = "~"
+set testdir = "~/cice_testing"
+set date0 = `date -u "+%y%m%d"`
+set date = ${date0}cc
+
+mkdir -p ${testdir}
+
+cd ${testdir}
+
+# Check out current cice master
+echo " "
+echo "*** checkout current cice master ***"
+git clone https://github.com/cice-consortium/cice cice.master.${date} --recursive
+cd cice.master.${date}
+set hash = `git show --oneline -s | cut -f 1 -d " "`
+cd ../
+
+# Check out test_cice_icepack, remove all code and copy from cice master
+# Need to be careful about dot files, particularly .git* files
+# This copies in all files via standard file expansion (except dot files at root)
+# This also copies in all dot file at the root that do not start with .g (ie. .git*)
+echo " "
+echo "*** checkout current test_cice_master ***"
+git clone https://github.com/apcraig/test_cice_icepack test_cice_icepack.${date}
```

Comment:
if the clone is done using SSH (`git@github.com:apcraig/Test_CICE_Icepack.git`) instead of HTTPS, the script will not ask for a password as long as the SSH keys are correctly configures

Reply:
Also, if we want to this to be easier to setup for others, I think the username (here `apcraig`) should be a script parameter, or at least an environment variable.

But as I wrote in https://github.com/CICE-Consortium/CICE/issues/89#issuecomment-616566377, I think it would be more "official" if the "test_cice_icepack" repo (or whatever we call it) was under the CICE-Consortium organization no ?

Reply:
Thanks @phil-blain.  We could use ssh, but I'm not sure it's worth setting up the ssh keys for occasional testing.  Putting in a username and password doesn't seem like a large burden or hurdle.

I have given a few people access to the test_cice_icepack repo.  You should be able to push to that repo using your own username/password.  Or are you suggesting that the "organization" name be a variable?  My idea is that a handful of us will have access to the repo under "apcraig", and we'll use it for now.  I don't see any value in setting up the equivalent repo under multiple users.  If we did that, we'd end up needing to test/change a few other things including the codecov location and the token string that connects github and codecov.  It seems just easier to give a subset of folks write access to my test_cice_icepack repo.  It's just a temporary repo and if it's get trashed, it's not a big deal.  Every use basically removes all files and adds them all back again anyway.

I replied in #89 that I would prefer to keep the test_cice_icepack repo hidden and out of the CICE-Consortium.  My hope is that it's temporary (until codecov supports submodules although I don't have a lot of optimism), and I prefer the fake repo not be visible to the community in general, especially since it's not part of the standard workflow and testing.  I think it can only create confusion in the community.  It would be easy to migrate test_cice_icepack to the consortium org if others felt it was better there though.